app/controllers/hosts_controller.rb

```diff
@@ -93,6 +93,7 @@ def new
   def clone
     @clone_host = @host
     @host = @clone_host.clone
+    @host.build = true
```

Comment:
Build is already copied without this PR, the issue is that when you clone a Host you would (normally) want `Build` to be true so that the host is actually built, otherwise it can be useless. 

@orrabin You made a good point about unmanaged hosts though, in that case we don't really care and it should be set to false. Otherwise we get an error (https://github.com/theforeman/foreman/blob/develop/app/models/host/base.rb#L507)

@timogoebel Can you add a check for that please?